submissions/migrations/0002_auto_20210223_0748.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.19 on 2021-02-23 13:48
+
+import jsonfield.fields
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('submissions', '0001_squashed_0005_CreateTeamModel'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='submission',
+            name='answer',
+            field=jsonfield.fields.JSONField(blank=True, db_column='raw_answer'),
```

Comment:
It seems odd to both be editing the previous migration and adding this one.  Can the parameters in the previous migration be updated so this one isn't needed?  I don't think there are any actual database changes.